components/CreateCollectiveMiniForm.js

```diff
@@ -62,6 +62,10 @@ const msg = defineMessages({
     id: 'Organization.Name',
     defaultMessage: 'Organization name',
   },
+  legalName: {
+    id: 'Organization.LegalName',
+    defaultMessage: 'Legal Name 🔒',
```

Comment:
```suggestion
    defaultMessage: 'Legal Name',
```

Can we rather use `PrivateInfoIcon`? Also ideally keep the icon out of the i18n string

Reply:
Makes sense. 👍🏽 I've changed that to use the `PrivateInfoIcon`. Let me know how it looks now.

Reply:
Did you try keeping the icon out of the i18n string? What was the blocker/incentive to have it in the string?

Reply:
Ops, when you meant keeping icon out of i18n string, I think I misunderstood a bit and thought it was to keep it outside the string but pass it as a value like the way I did. Sorry. But now I see you pushed that change. Thanks a lot. 🙂